lib/tapjoy/ldap/cli/group/remove_user.rb

```diff
@@ -0,0 +1,37 @@
+module Tapjoy
+  module LDAP
+    module CLI
+      module Group
+        # Remove existing user to existing group
+        class RemoveUser
+          def remove_user
+            confirm unless opts[:force]
+            puts Tapjoy::LDAP::API::Group.update(
+              opts[:group], opts[:username], :delete)
+          end
+
+          private
+          def opts
+            @opts ||= Trollop.options do
+              # Set help message
+              usage 'group remove_user [options]'
+              synopsis "\nThis command is for removing existing users from existing groups"
+
+              opt(:group, 'Specify group', :type => :string, :required => true)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use the new Ruby 1.9 hash syntax.](https://www.codacy.com/app/ali-tayarani/ldap_tools/file/2634528070/issues/source?bid=3316970&fileBranchId=3318857#l20)